integration-tests/quartz/src/main/java/io/quarkus/it/quartz/DisabledMethodResource.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.it.quartz;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("/scheduler/disabled")
+public class DisabledMethodResource {
+
+    @Inject
+    DisabledScheduledMethods disabledScheduledMethods;
+
+    @GET
+    @Path("cron")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getCronCount() {
+        return disabledScheduledMethods.valueSetByCronScheduledMethod;
```

Comment:
You shouldn't access a static variable like this...

Reply:
I didn't give this much thought since it is a variable that's there to only show that it is not initialized.